README.md

```diff
@@ -89,17 +89,15 @@ Thanks!
 
 # How to use Soot's Java 9 Features?
 
-If you want to run Soot with Java > 8, you are done. Just run it as usal.
-If you want to execute Soot with Java 8 but analyze Java >8 Projects or vice versa, see below.
-
 ## Use from Source Code
 To load modules in Soot's `ModuleScene` from java:
 ```.java
-// configure Soot's options
+// configure Soot's options, refer to example configurations below
 Options.v().set_soot_modulepath(modulePath);
 
 
 // load classes from modules into Soot
+// Here, getClassUnderModulePath() expects the module path to be set using the Options class
 Map<String, List<String>> map = ModulePathSourceLocator.v().getClassUnderModulePath(modulePath);
 for (String module : map.keySet()) {
    for (String klass : map.get(module)) {
```

Comment:
let's explain below where we have the call to loadClass() that it's defined just below